front/package-lock.json

```diff
@@ -4,7 +4,6 @@
   "requires": true,
   "packages": {
```

Comment:
Not sure why this is happening. On main there is no "front", I guess I need to rebase, but since this PR is based on another PR, I will wait for the other one to merge before adjusting it. You can review the rest of the code for now.